dev-packages/cli/src/commands/update-next.ts

```diff
@@ -0,0 +1,119 @@
+/********************************************************************************
+ * Copyright (c) 2024 EclipseSource and others.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License v. 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0.
+ *
+ * This Source Code may also be made available under the following Secondary
+ * Licenses when the conditions for such availability set forth in the Eclipse
+ * Public License v. 2.0 are satisfied: GNU General Public License, version 2
+ * with the GNU Classpath Exception which is available at
+ * https://www.gnu.org/software/classpath/license.html.
+ *
+ * SPDX-License-Identifier: EPL-2.0 OR GPL-2.0 WITH Classpath-exception-2.0
+ ********************************************************************************/
+
+import * as fs from 'fs';
+import { glob } from 'glob';
+import * as jq from 'node-jq';
+import * as path from 'path';
+import * as sh from 'shelljs';
+import { baseCommand, configureShell } from '../util/command-util';
+import { LOGGER, configureLogger } from '../util/logger';
+import { validateGitDirectory } from '../util/validation-util';
+export const UpdateNextCommand = baseCommand() //
+    .name('updateNext')
+    .alias('u')
+    .description('Updates all `next` dependencies in GLSP project to the latest version')
+    .argument('[rootDir]', 'The repository root', validateGitDirectory, process.cwd())
+    .option('-v, --verbose', 'Enable verbose (debug) log output', false)
+    .action(updateNext);
+
+export async function updateNext(rootDir: string, options: { verbose: boolean }): Promise<void> {
+    configureLogger(options.verbose);
+    configureShell({ silent: false, fatal: true });
+
+    LOGGER.info('Updating next dependencies ...');
+    rootDir = path.resolve(rootDir);
+    const packages = await getWorkspacePackages(rootDir);
+    LOGGER.debug(`Scanning ${packages.length} packages to derive resolutions`, packages);
+    const resolutions = await getResolutions(packages);
+    if (!resolutions) {
```

Comment:
```suggestion
    if (!resolutions || Object.keys(resolutions).length === 0) {
```

`getResolutions` return an empty record as well.